py3dtilers/Common/group.py

```diff
@@ -152,43 +155,42 @@ def distribute_objects_in_polygons(self, feature_list, polygons):
         :param polygons: a list of Shapely polygons
         """
 
-        objects_to_tile_dict = {}
-        objects_to_tile_without_poly = {}
+        features_dict = {}
+        features_without_poly = list()
 
         # For each feature, find the polygon containing it
         for i, feature in enumerate(feature_list):
             p = Point(feature.get_centroid())
             in_polygon = False
             for index, polygon in enumerate(polygons):
                 if p.within(polygon):
-                    if index in objects_to_tile_dict:
-                        objects_to_tile_dict[index].append(i)
-                    else:
-                        objects_to_tile_dict[index] = [i]
+                    if index not in features_dict:
+                        features_dict[index] = []
+                    features_dict[index].append(i)
                     in_polygon = True
                     break
             if not in_polygon:
-                objects_to_tile_without_poly[i] = [i]
+                features_without_poly.append(i)
 
         # Create a list of Group
         groups = list()
-        for key in objects_to_tile_dict:
-            additional_points = polygons[key].exterior.coords[:-1]
-            contained_objects = FeatureList([feature_list[i] for i in objects_to_tile_dict[key]])
-            group = Group(contained_objects, with_polygon=True, additional_points=additional_points)
+
+        for key in features_dict:
+            polygon = polygons[key].exterior.coords[:-1]
+            contained_features = FeatureList([feature_list[i] for i in features_dict[key]])
+            group = Group(contained_features, polygons=[polygon])
             groups.append(group)
-        for key in objects_to_tile_without_poly:
-            contained_objects = FeatureList([feature_list[i] for i in objects_to_tile_without_poly[key]])
-            group = Group(contained_objects)
+
+        for feature_index in features_without_poly:
+            group = Group(FeatureList([feature_list[feature_index]]))
             groups.append(group)
 
-        return self.distribute_groups_in_cubes(groups, 300)
+        return self.distribute_groups_in_cubes(groups, 1000)
```

Comment:
- Added [default cube size](https://github.com/VCityTeam/py3dtilers/commit/aa2505d00c457f2b8a304461f06747fdb57b93f5)
- [issue](https://github.com/VCityTeam/py3dtilers/issues/76) about kd_tree_max. This will be done after this PR